pkg/gofr/datasource/file/ftp/file.go

```diff
@@ -244,29 +240,18 @@ func (f *ftpFile) Check(whence int, offset, length int64) (int64, error) {
 
 // Seek sets the offset for the next Read or ReadAt operation.
 func (f *ftpFile) Seek(offset int64, whence int) (int64, error) {
-	var (
-		err error
-		r   ftpResponse
-	)
-
-	r, err = f.conn.Retr(f.path)
+	n, err := f.conn.FileSize(f.path)
 	if err != nil {
+		f.logger.Errorf("Seek", "Failed", "Error : %v", err)
 		return 0, err
 	}
 
-	p, err := io.ReadAll(r)
+	res, err := f.check(whence, offset, n)
 	if err != nil {
-		return 0, err
+		f.logger.Errorf("Seek", "Failed", "Error : %v", err)
```

Comment:
You have removed the `return` here.

This means, we will get a `Seek Failed Error:` message followed by a `Seek Success Offset at whence` message.